apps/admin-gateway/src/module.api/orders.controller.ts

```diff
@@ -124,7 +124,7 @@ export class OrdersController {
       streamUpdates: true
     }
     return await lastValueFrom<ResponseWithStatus>(
-      this.ordersClient.send(QUEUE_MESSAGE.UPDATE_ORDER_STATUS, payload).pipe(
+      this.ordersClient.emit(QUEUE_MESSAGE.UPDATE_ORDER_STATUS, payload).pipe(
```

Comment:
Incorrect. Send is the correct method. refer to line `69` in `order-service.controller.ts`

Reply:
`send` returns 500.

Reply:
`send` returns 500.

Reply:
Even so but that doesnt solve the underlying issue. Check if anything is getting returned

Reply:
![image](https://github.com/user-attachments/assets/6f7ef5c0-f6b8-4898-9a00-e327895b1e01)


Wrap the code in the circle in an observable. `lastValueFrom`

Reply:
Please learn to look at for root cause when debugging. If something doesnt make sense and it works, there is something definitely wrong

Reply:
The request body still reaches the orders controller if `send` or `emit` is used. The error is only thrown from the admin gateway. 
> Test with and without the `lastValueFrom` observable